doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,106 +3,215 @@
+
+Steps to reproduce:
+
+. Create a project with a process.
+. Deploy the project and start the process.
+. Go to the *Execution Servers* page and stop the container.
+
+Actual result: The container is removed from the UI even though it is still running.
+ 
+ifdef::PAM[]
+== Maven repository
+.Missing JWS dependencies in offline maven repo distribution [https://issues.jboss.org/browse/RHPAM-1715[RHPAM-1715]]
+
+Issue: The following key dependencies required to install {KIE_SERVER} on {JWS} are missing from the offline Maven repository. 
 
 [source]
 ----
-An unexpected error happened. Please try again in a few moments.
+org.jboss.integration:narayana-tomcat
+org.jboss:jboss-transaction-spi
 ----
 
-endif::[]
+If you are you are performing a manual installation, you must copy the files described in the workaround.
 
-== Decision engine
-.MVEL expression (1 + 2 * 3 + $v ) causes a "no such method or function " error for bind variable[https://issues.jboss.org/browse/RHDM-707[RHDM-707]]
+Workaround:
 
-Issue: When using an expression such as `(1 + 2 * 3 + $v)` in the consequence of a rule which uses the mvel dialect, where `$v` is a bound variable from rule constraints, the evaluation of the rule produces an exception.
+. Use the installer to install {PRODUCT} on {JWS}.
+. In a terminal, change directory to the `<JWS-HOME>/jws-5.0/tomcat/lib` directory where the installer installed {PRODUCT}.
+. Copy the following files from the `<JWS-HOME>/jws-5.0/tomcat/lib` directory to the `<JWS-HOME>/jws-5.0/tomcat/lib` directory of the target {JWS} location.
++
+[source]
+----
+narayana-tomcat-1.0.0.Final-redhat-1.jar 
+jboss-transaction-spi-7.6.0.Final-redhat-1.jar 
+----
+endif::PAM[]
 
-Workaround: Change the expression to `(1 + (2 * 3) + $v)`.
+== OpenShift
+.Various errors occur when starting clustered {CENTRAL} on OpenShift [https://issues.jboss.org/browse/RHPAM-1747[RHPAM-1747]]
+
+Issue: When clustered {CENTRAL} is started on OpenShift using the `rhpam72-authoring-ha.yaml` template, {CENTRAL} sometimes throws various exceptions that cause the pod to restart or the deployment to fail.
+
+Steps to reproduce:
+
+. Use the `rhpam72-authoring-ha.yaml` template to deploy {PRODUCT} {PRODUCT_VERSION} images on OpenShift.
+. Check the {CENTRAL} log files.
+
+Expected result: There are no errors in the log files.
+
+Actual result: Errors appear in the log files, the pod to restarts, or the deployment fails.
+
+Workaround: None.
 
 ifdef::PAM[]
 == Process designer
+.In the new process designer, when a process containing another event subprocess is created and saved, the process validation fails [https://issues.jboss.org/browse/RHPAM-1732[RHPAM-1732]]
+
+Issue: In the new process designer, when a subprocess containing another event subprocess is created and saved, the process validation fails. 
+
+[NOTE]
+====
+Because of this issue, the new process designer does not let you add an event subprocess into another (embedded) subprocess.
```

Comment:
"As a result of this issue, you cannot add an event subprocess into another (embedded) subprocess in the new process designer."